package.json

```diff
@@ -6,9 +6,16 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "eslint-plugin-prettier": "^4.0.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^8.0.2",
+    "react-router-dom": "^6.3.0",
     "react-scripts": "4.0.3",
+    "redux": "^4.2.0",
```

Comment:
I highly recommend you to use [Redux Toolkit](https://redux-toolkit.js.org/) instead of the regular redux, the reason is that a lot of logic, configurations and repetitive stuff is already under the hood with this library.